npm/src/typings.ts

```diff
@@ -135,9 +135,45 @@ export interface SAMLProfile {
 export interface JacksonOption {
   externalUrl: string;
   samlPath: string;
-  samlAudience?: string;
+  samlAudience: string;
```

Comment:
No, it was an optional type, should change it back @devkiran

Reply:
but how can it be optional?